examples/run_over_exo.sh

```
@@ -0,0 +1,68 @@
+        return 1
+    fi  
+
+    curl -sf -X POST http://localhost:52415/instance \
+        -H "Content-Type: application/json" \
+        -d "{\"instance\": ${placement}}" >/dev/null
+
+    # Wait for instance to be ready
+    echo -n "Waiting for model to load"
+    retries=60
+    while (( retries-- > 0 )); do
+        local result
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)